src/Components/NewComment.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from "react";
+import { postAComment } from "../assets/Requests/api";
+
+const NewComment = ({ article_id, setComments }) => {
+  const [comment, setComment] = useState("");
+  const [username, setUsername] = useState("jessjelly");
+
+  useEffect(() => {
+    setUsername("jessjelly");
+  }, []);
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postAComment(article_id, username, comment).then((newComment) => {
+      setComments((currComments) => {
```

Comment:
this is fine, and gives enough feedback with the comment appearing at the top after pressing post - though optimistic rendering might be preferable in circumstances where there may be a lot of people leaving comments while a single user is still writing their comment.